docs/feature-gates.md

```diff
@@ -36,6 +36,7 @@ example, to enable `AntreaProxy` on Linux, edit the Agent configuration in the
 | `AntreaProxy`           | Agent              | `false` | Alpha | v0.8.0        | N/A          | N/A        | Yes                | Must be enabled for Windows. |
 | `ClusterNetworkPolicy`  | Controller         | `false` | Alpha | v0.8.0        | N/A          | N/A        | No                 |       |
```

Comment:
oh damn.. this needs to be updated.. CNP now needs Agent side flag too :(

Reply:
please open a new PR for this, I will merge this one